lib/generators/hyrax/templates/mediated_deposit_workflow.json.erb

```diff
@@ -58,6 +58,12 @@
                             "to": ["approving"]
                         }
                     ]
+                }, {
+                    "name": "leave_a_comment",
```

Comment:
Are there any states that come from this state?  I'm not sure why this should be a terminal state?